apps/cowswap-frontend/src/modules/bridge/components/BridgeRouteBreakdown.tsx

```diff
@@ -203,9 +297,20 @@ export function BridgeRouteBreakdown({
     : undefined
   const { value: bridgeReceiveAmountUsd } = useUsdAmount(bridgeReceiveAmountCurrency)
 
-  return (
-    <Wrapper>
-      <RouteHeader>
+  // Handle header click for accordion functionality - triggers the callback to toggle expanded state
+  const handleHeaderClick = () => {
+    if (isCollapsible) {
+      onExpandToggle()
+    }
+  }
+
+  // Renders the route header - shared between collapsed and expanded views to avoid code duplication
+  const renderHeader = () => {
```

Comment:
Please, never use render functions:
https://dev.to/adevnadia/react-re-renders-guide-preventing-unnecessary-re-renders-21dm?comments_sort=oldest

Preventing re-renders with composition
⛔️ Antipattern: Creating components in render function

Reply:
Addressed